testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -9504,8 +9504,10 @@ fn test_problematic_blocks_are_not_relayed_or_stored() {
 
     let tip_info = get_chain_info(&conf);
 
-    // all blocks were processed
-    assert!(tip_info.stacks_tip_height >= old_tip_info.stacks_tip_height + 5);
+    // at least one block was mined (hard to say how many due to the raciness between the burnchain
+    // downloader and this thread).
+    assert!(tip_info.stacks_tip_height > old_tip_info.stacks_tip_height);
+
```

Comment:
There's a race condition in `next` at least (and probably also `develop` and `master`) whereby the burnchain block downloader thread races the miner thread, such that a block-commit generated by the miner thread is not mined in the target block.  This, in turn, causes a sortition to be missed, which in turn causes this assertion to fail since the Stacks height would have advanced by 4 blocks instead of 5.  However, the nature of this test is to verify that a block with a problematic transaction gets mined (which happens below); it doesn't matter if it takes 4 or 5 attempts.  This check was needlessly strict.

I fixed it here because I earlier approved a PR to `next` where this test was failing for the same reason, and I told the author not to worry about it since it wasn't blocking the merge (and the code changes were unrelated).